src/Screens/ChatScreen.tsx

```diff
@@ -17,8 +17,16 @@ import Loading from '../Components/loading';
 import Colors from '../Contants/Colors';
 import {launchCamera, launchImageLibrary} from 'react-native-image-picker';
 import storage from '@react-native-firebase/storage';
-import {Image, LogBox, StyleSheet, Text, TouchableOpacity, View} from 'react-native';
-import moment from 'moment';
+import {
+  Alert,
```

Comment:
"Since you are not using the 'alert' function in this file, please consider removing the import statement related to it."